lib/rubocop/cop/rails/action_name.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Use only specific action names.
```

Comment:
If the background of this PR description is written, the intention will be conveyed to users.